packages/core/src/__tests__/prefetch.js

```diff
@@ -0,0 +1,149 @@
+'use strict'
```

Comment:
I don't get why this test should go into the the test suite?

Reply:
If this is an example, please move it in examples or documentation.

Reply:
Yes, example, didn't plan to keep it in, just to challenge / discuss the issue.

Reply:
@ognen do you want to move it to example tests, or just a create a markdown doc page in `docs`?